icons/scissors.svg

```diff
@@ -10,8 +10,8 @@
   stroke-linejoin="round"
 >
   <circle cx="6" cy="6" r="3" />
+  <path d="M8.2 8.2 12 12" />
+  <path d="M14.8 14.8 20 20" />
   <circle cx="6" cy="18" r="3" />
-  <line x1="20" x2="8.12" y1="4" y2="15.88" />
-  <line x1="14.47" x2="20" y1="14.48" y2="20" />
-  <line x1="8.12" x2="12" y1="8.12" y2="12" />
+  <path d="M20 4 8.2 15.8" />
```

Comment:
There seems to be an issue with optimisation:

Original outline:
![image](https://github.com/lucide-icons/lucide/assets/17746067/f7acaf9e-80b5-4495-a81c-87af4c36957a)

New outline:
![image](https://github.com/lucide-icons/lucide/assets/17746067/a371be0c-f412-4ebe-957f-59a10e41d852)
(note gaps)